hvpy/facade.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def _copy_docstring(input_class):
+    def decorator(func):
+        func.__doc__ = (
+            "\n".join(func.__doc__.split("\n")[1:3])
+            + "\n".join(input_class.__doc__.split("\n")[2:])
+            + "\n".join(func.__doc__.split("\n")[2:])
+        )
+        return func
+
+    return decorator
+
+
+@_copy_docstring(getJP2ImageInputParameters)
+def getJP2Image(
+    date: datetime,
+    sourceId: int,
+    jpip: bool = False,
+    json: bool = False,
+):
+    """
+    Retrieve a JP2000 image from the helioviewer.org API.
+
+    Example:
+
+        >>> getJP2Image(date=datetime(2019,1,1), sourceId=1, jpip=True)
```

Comment:
This example will want the imports required as well. 

Ideally I should be able to copy and paste an example into a ipython terminal and have it run.